src/cogs/trivia.py

```diff
@@ -4,7 +4,9 @@
 from asyncio import sleep, TimeoutError
 from Levenshtein import ratio
 
+
 locked_channels = []  # Channel IDs that are currently in use by a game
+available_channels = []
```

Comment:
I think I figured out your linter error. What initializes this array when the bot runs for the first time, and values are already set in the config file? From the looks of this, nothing.